StartEycaActivationOrchestrator/__tests__/index.test.ts

```diff
@@ -0,0 +1,109 @@
+// tslint:disable: object-literal-sort-keys
+
+import { FiscalCode } from "italia-ts-commons/lib/strings";
+import { context as contextMock } from "../../__mocks__/durable-functions";
+import { handler } from "../index";
+
+const aFiscalCode = "RODFDS82S10H501T" as FiscalCode;
+
+const getInputMock = jest.fn();
```

Comment:
We can use to avoid duplicated configuration for happy path tests
```suggestion
const getInputMock = jest.fn().mockImplementation(() => ({
      fiscalCode: aFiscalCode
    }));
```